starknet_py/tests/e2e/fixtures/proxy.py

```diff
@@ -71,9 +75,26 @@ async def deploy_proxy_to_contract_custom(
 
 
 @pytest_asyncio.fixture(
-    params=[("oz_proxy_exposed_compiled.json", "map_compiled.json")]
+    name="proxy_implementation_func",
+    params=[("oz_proxy_implementation_func_compiled.json", "map_compiled.json")],
+)
+async def deploy_proxy_to_contract_implementation_func(
+    request, new_gateway_account_client: AccountClient
+) -> DeployResult:
+    """
+    Declares a contract and deploys a custom proxy pointing to that contract.
+    """
+    compiled_proxy_name, compiled_contract_name = request.param
+    return await deploy_proxy_to_contract(
+        compiled_proxy_name, compiled_contract_name, new_gateway_account_client
+    )
+
+
+@pytest_asyncio.fixture(
+    name="proxy_impl_func",
+    params=[("oz_proxy_impl_func_compiled.json", "map_compiled.json")],
 )
-async def deploy_proxy_to_contract_exposed(
+async def deploy_proxy_to_contract_impl_func(
```

Comment:
Why was this renamed?

Reply:
In order to show the different `CustomProxyCheck` in docs guide. The old custom proxy check behaved similarly to how the eth proxy check behaves now.